app/__init__.py

```diff
@@ -3,5 +3,8 @@
 
 def create_app(test_config=None):
     app = Flask(__name__)
-
+    
+    from .routes import planets_bp    
+    app.register_blueprint(planets_bp)
+    
```

Comment:
👍 Looks good! We can also move this file into a routes folder which would change the path of this import.

Reply:
> 👍 Looks good! We can also move this file into a routes folder which would change the path of this import.

Hi Audrey, we don't understand what you mean. Could you please explain it once more? Thank you ! :)